src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,22 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+public class AuthenticationService extends UserService {
```

Comment:
```suggestion
public class AuthenticationService {
```
you don't need to do this, you should create an instance of UserService inside of the login method and call findByEmail using it